### DISCUSSION TITLE
Layering curves

### DISCUSSION THREAD

**Discussion by SilentKnightED** [WhiteMagic/JoystickGremlin#535] [Q&A]
It would be extremely helpful if we could apply a curve to individual axis when merging.
For example:
1. x-axis of right stick -> set desired curve
2. x-axis of right MINIstick -> set desired curve
3. Merge 1/2 above
4. Merged axis can have another curve applied if needed. 

The critical points are 1-3. It does not make sense to apply a curve to the merged axis otherwise, as you are dealing with different physical inputs. In the example above, obviously my curve for a ministick is going to be very different than my curve for a full joystick. This ::should:: not be technically a problem. It would be like applying an effect to a photoshop layer, then grouping them, and applying another (or same) effect to the group.

**Comment by WhiteMagic** [comment]
This is not something you can do with the UI as the merge action is decoupled from the other actions. This is something R14 supports but in R13 the only way to achieve that is to write a plugin that does these things via code.
